application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Macro.xml

```diff
@@ -665,7 +660,33 @@ if (typeof(XWiki) == "undefined" || typeof(XWiki.widgets) == "undefined" || type
   }) // end PlainList extends View
 
   jQuery.fullCalendar.views.plainList = PlainList;
-  
+  /**
+  * Events triggered before deleteEvents modal is shown: save the button that triggers
+  * the modal, close MoccaCalendarPopup, update the name of event on displayed text.
+  */
+  jQuery(document).on('show.bs.modal', '#deleteEvents', function(event) {
+    var relatedTarget = $(event.relatedTarget);
```

Comment:
Isn't ``$`` referring to Prototype.js here? Be careful when mixing jQuery and Prototype.js . Can't you put this code in a require block?